index.js

```diff
@@ -1,41 +1,74 @@
-    if (arg === null) {
-      for (let i = 0; i < Books.length; i++) {
+class Library {
+  books;
+
+  constructor() {
+    this.books = [];
+  }
+
+  display(initialObj, arg) {
+    if (arg === null && initialObj !== null) {
+      for (let i = 0; i < initialObj.length; i += 1) {
         document.getElementById('books').innerHTML += `
-					<h3>${this.books[i].author}</h3>
-					<p>${this.books[i].title}</p>
-					<button id='remove' onClick='bi.remove(${i})'>Remove</button>
-					<hr>
-					`;
+            <li class = "bck">
+              <div class ="main">
+              <p>${initialObj[i].author}</p>
+              <p>by ${initialObj[i].title}</p>
+              </div>
+              <button id='remove' onClick='bi.remove(${i})'>Remove</button>
+            </li>
+            `;
+      }
+    } else if (arg === null) {
+      localStorage.setItem('datas', JSON.stringify(this.books));
+      document.getElementById('books').innerHTML = '';
+      for (let i = 0; i < JSON.parse(localStorage.getItem('datas')).length; i += 1) {
+        document.getElementById('books').innerHTML += `
+            <li class = "bck">
+              <div class ="main">
+              <p>${JSON.parse(localStorage.getItem('datas'))[i].author}</p>
+              <p> by ${JSON.parse(localStorage.getItem('datas'))[i].title}</p>
+              </div>
+              <button id='remove' onClick='bi.remove(${i})'>Remove</button>
+            </li>
+            `;
       }
     } else {
+      localStorage.setItem('datas', JSON.stringify(this.books));
       document.getElementById('books').innerHTML = '';
       for (let i = 0; i < arg.length; i += 1) {
         document.getElementById('books').innerHTML += `
-						<h3>${this.books[i].author}</h3>
-						<p>${this.books[i].title}</p>
-						<button id='remove' onClick='bi.remove(${i})'>Remove</button>
-						<hr>
-						`;
+        <li class = "bck">
+          <div class ="main">
+            <p>${arg[i].author}</p>
+            <p>by ${arg[i].title}</p>
+            </div>
+            <button id='remove' onClick='bi.remove(${i})'>Remove</button>
+          </li>
+            `;
       }
     }
-  };
+  }
 
-  this.remove = function (ref) {
+  remove(ref) {
+    this.books = JSON.parse(localStorage.getItem('datas'));
     this.books.splice(ref, 1);
-    this.display(this.books);
-  };
+    localStorage.setItem('datas', JSON.stringify(this.books));
+    this.display(null, JSON.parse(localStorage.getItem('datas')));
+    this.books = JSON.parse(localStorage.getItem('datas'));
+  }
 
-  this.add = function (author, title) {
+  add(author, title) {
     const hauteur = document.getElementById(author).value;
     const titre = document.getElementById(title).value;
-
-    this.books.push({ author: hauteur, title: titre });
-    this.display(this.books);
-  };
+    if (titre !== '' && hauteur !== '') {
+      this.books = JSON.parse(localStorage.getItem('datas'));
+      this.books.push({ author: hauteur, title: titre });
```

Comment:
# Optional:  Please use `concat()` methods because it doesn't mutate the array
- The `push()` method is a generic method similar to call() or apply(). It will mutate your array (or object), pushing a new value into it.
- The `concat()` method returns a new array, with the values merged. This also avoids mutation side effects.
- [Read more here.](https://stackoverflow.com/questions/48428044/why-use-concat-instead-of-push-in-this-instance)